PhysicsTools/NanoAOD/python/jetsPAIReD.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+from  PhysicsTools.NanoAOD.common_cff import *
+
+pairedJetTable = cms.EDProducer("PAIReDONNXJetTagsProducer")
```

Comment:
looks like you are using all default values from the fillDescription, which. is actually good to have. but I'd rather, so that the actual parameters are visible (and modifiable) in the configuration, that you import from the default cfi generated from the fillDescription (See below)

Reply:
Do you mean adding a file similar to this one https://cmssdt.cern.ch/lxr/source/cfipython/RecoBTag/ONNXRuntime/UnifiedParticleTransformerAK4ONNXJetTagsProducer.py that specifies the descriptions outside of the producer? Or if not is there by any chance a different piece of code you could point me to that accomplishes something similar to what you would like to see here?

Reply:
adding https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp for documentation. you do not have to add the file in cfipython yourself, the fwk will do this for your upon "scram build". You should be able to do something like
```
from RecoBTag.ONNXRuntime.pAIRrDONNXJetTagsProducer_cfi import pAIRrDONNXJetTagsProducer
pairedJetTable = pAIRrDONNXJetTagsProducer.clone()
```

Reply:
`from RecoBTag.ONNXRuntime.pAIRrDONNXJetTagsProducer_cfi import pAIRrDONNXJetTagsProducer` is what `descriptions.addWithDefaultLabel(desc);` provides